tst/EnergyPlus/unit/SimAirServingZones.unit.cc

```diff
@@ -1568,4 +1569,530 @@ TEST_F(EnergyPlusFixture, InitAirLoops_2AirLoop3ADUb)
+
+        "SetpointManager:Scheduled,",
+        "  Scheduled Deck Temp,                    !- Name",
+        "  Temperature,                            !- Control Variable",
+        "  Deck Temperature,                       !- Schedule Name",
+        "  Supply Side Outlet Node;                !- Setpoint Node or NodeList Name",
+
+        "Schedule:Constant,",
+        "  Deck Temperature,                       !- Name",
+        "  Any Number,                             !- Schedule Type Limits Name",
+        "  14;                                     !- Hourly Value",
+
+        "AirLoopHVAC:SupplyPath,",
+        "  Air Loop HVAC Demand Inlet Node Supply Path, !- Name",
+        "  Demand Inlet Node,                      !- Supply Air Path Inlet Node Name",
+        "  AirLoopHVAC:ZoneSplitter,               !- Component Object Type 1",
+        "  Air Loop HVAC Zone Splitter;            !- Component Name 1",
+
+        "AirLoopHVAC:ZoneSplitter,",
+        "  Air Loop HVAC Zone Splitter,            !- Name",
+        "  Demand Inlet Node,                      !- Inlet Node Name",
+        "  ATU VAV No Reheat Inlet Node;           !- Outlet Node Name 1",
+
+        "AirLoopHVAC:ReturnPath,",
+        "  Air Loop HVAC Return Path,              !- Name",
+        "  Demand Outlet Node,                     !- Return Air Path Outlet Node Name",
+        "  AirLoopHVAC:ZoneMixer,                  !- Component Object Type 1",
+        "  Air Loop HVAC Zone Mixer;               !- Component Name 1",
+
+        "AirLoopHVAC:ZoneMixer,",
+        "  Air Loop HVAC Zone Mixer,               !- Name",
+        "  Demand Outlet Node,                     !- Outlet Node Name",
+        "  Zone1 Return Air Node;                  !- Inlet Node Name 1",
+
+        "Output:Variable,",
+        "  *,                                      !- Key Value",
+        "  System Node Mass Flow Rate,             !- Variable Name",
+        "  Detailed;                               !- Reporting Frequency",
+
+        "Output:SQLite,",
+        "  SimpleAndTabular;                       !- Option Type",
+
+    });
+
+    ASSERT_TRUE(process_idf(idf_objects));
+
+    SimulationManager::ManageSimulation(*state); // run the design days
+
+    int returnFanNode = UtilityRoutines::FindItemInList("VSD RETURN FAN OUTLET TO MIXING BOX NODE", DataLoopNode::NodeID, DataLoopNode::NumOfNodes);
+    EXPECT_GT(returnFanNode, 0);
+    int supplyOutletNode = UtilityRoutines::FindItemInList("SUPPLY SIDE OUTLET NODE", DataLoopNode::NodeID, DataLoopNode::NumOfNodes);
+    EXPECT_GT(returnFanNode, 0);
+    EXPECT_GT(supplyOutletNode, 0);
+
+    EXPECT_EQ(0, DataLoopNode::Node(returnFanNode).MassFlowRateMin);
+    EXPECT_EQ(0, DataLoopNode::Node(supplyOutletNode).MassFlowRateMin);
+    EXPECT_EQ(0, DataLoopNode::Node(returnFanNode).MassFlowRate);
+    EXPECT_EQ(0, DataLoopNode::Node(supplyOutletNode).MassFlowRate);
```

Comment:
Ensures that the flow rate is indeed zero, and not corresponding to 0.3 Min flow frac